cypress/support/step_definition/course/commonCourseRelatedSteps.spec.js

```diff
@@ -0,0 +1,38 @@
+import Courses_Common from '../../pages/course/pageCommonCourses'
+import Courses from '../../pages/course/pageCourses'
+
+const coursesCommon = new Courses_Common()
+const courses = new Courses()
+
+When('I go to rooms overview', () => {
+  coursesCommon.navigateToRoomsOverview()
+})
+
+And('I go to room {string}', (room_name) => {
+  const selectedRoom = `[aria-label='${room_name}']`
+  cy.get(selectedRoom).click({ multiple: true, force: true })
```

Comment:
Shouldn't this call "navigateToRoomBoard"?

Reply:
done!